tests/acceptance/test_performance_summary.py

```diff
@@ -204,7 +205,5 @@ def test_transaction_threshold_modal(self, mock_now):
                 '[data-test-id="grid-editable"] [data-test-id="empty-state"]', timeout=2
             )
             # We have to wait for this again because there are loaders inside of the table
-            self.page.wait_until_loaded()
```

Comment:
Are these additional waits no longer necessary? If not can we remove the corresponding comment?

Reply:
this one was an accidental extra wait, and the ones below are no longer necessary since opening the modal doesn't have any async stuff-- will remove 👍

Reply:
Oh I think this one I don't care about since I don't care about the stuff in the table loading. Although maybe I should add it back to prevent flakey tests